script/ci_build.sh

```diff
@@ -123,6 +123,32 @@ then
     MSBUILD="/$MSBUILD/MSBuild/Current/Bin/MSBuild.exe"
 fi
 
+find_executables() {
```

Comment:
It's confusing that function "find_something" actually copies something. Consider renaming.